src/Pages/AddRide/AddRide.jsx

```diff
@@ -0,0 +1,227 @@
+import './AddRide.css'
+import { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { addRide } from '../../redux/slices/addRideSlice';
+import { useNavigate } from 'react-router-dom';
+import { loginUser } from "../../redux/slices/loginSlice.js";
+import Dialog from '../../components/Dialog/Dialog';
+
+
+const AddRide = () => {
```

Comment:
- [ ] For some reason, after creating a user, the app is acting as if I'm not logged in and won't let me add a new car:

![image](https://github.com/EddyShimwa/Rent-A-Ride/assets/57421823/ccc658d3-deb3-493a-b348-4e9db126be52)

Kindly correct this functionality so your app works as intended 💪

Reply:
I just created a new user, and then proceeded to log in, and added a new car and it worked as expected, please. 

This is the new data I just added:
![new-data](https://github.com/EddyShimwa/Rent-A-Ride/assets/32777408/038d7671-d4a1-4bd9-a8f4-8607be81a46f)

This is the console response to my latest action. i,e user account creation, user log in and new car addition:

![rails-console](https://github.com/EddyShimwa/Rent-A-Ride/assets/32777408/b763160b-7562-48f7-a307-12245f5418d6)